docs/graphql/core/getting-started/first-event-trigger.rst

```diff
@@ -25,6 +25,8 @@ Create a table
 Head to the Hasura console, navigate to ``Data -> Create table`` and create a sample table called ``profile`` with
 the following columns:
 
+Let's say we have the following table:
```

Comment:
We already introduce this by saying "create a table called...". We can remove this.